Octokit.Tests.Conventions/ModelTests.cs

```diff
@@ -98,12 +99,10 @@ public void ResponseModelsHaveGetterOnlyProperties(Type modelType)
             {
                 var setter = property.GetSetMethod(nonPublic: true);
 
-                if (setter == null || !setter.IsPublic)
+                if (setter != null && (setter.IsPublic || typeof(RepositoryPermissions) == modelType))
```

Comment:
After changing all the auto-props to get-only this becomes
```suggestion
                if (setter != null)
```
```
```